api/src/controllers/article.js

```diff
@@ -7,40 +7,45 @@ import logger from '../utils/logger';
 
 exports.list = async (req, res) => {
 	const query = req.query || {};
+
 	let articles = [];
+
 	if (query.type === 'recommended') {
 		articles = await getArticleRecommendations(req.User);
 	} else {
 		if (query.rss && !mongoose.Types.ObjectId.isValid(query.rss)) {
 			return res.status(400).json({ error: `Invalid RSS id ${query.rss}` });
 		}
+
 		articles = await Article.apiQuery(req.query);
 	}
 
 	res.json(articles.filter(a => a.valid));
 };
 
 exports.get = async (req, res) => {
-	let articleID = req.params.articleId;
+	let articleId = req.params.articleId;
```

Comment:
whats the correct style in Node?

Reply:
@tschellenbach It's a personal preference to be honest. I do `Id` and others do `ID`.